sqlalchemy_redshift/dialect.py

```diff
@@ -604,6 +604,11 @@ def _get_table_or_view_names(self, relkind, connection, schema=None, **kw):
     def _get_column_info(self, *args, **kwargs):
         kw = kwargs.copy()
         encode = kw.pop('encode', None)
+        if sa.__version__ >= '1.2.0':
```

Comment:
is there a distutils.version class we can use here?

Reply:
How about a little bit of magic:

```python
try:
    from inspect import getfullargspec as getargspec
except ImportError:
    from inspect import getargspec

if 'comment' in getargspec(super(RedshiftDialect, self)._get_column_info).args:
    ...
```

Reply:
Yo dawg I heard you like reflection

Reply:
Well that's fancy. Will try this.

Reply:
I strongly recommend against this. Reflection in Python is a nasty thing to introduce to production code, both for performance and predictability reasons. Checking `__version__` is a very fine way to achieve what you're doing.

Reply:
I've gone back to this simpler version check.